src/app/(proper_react)/(redesign)/Shell/Shell.test.tsx

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { it, expect } from "@jest/globals";
+import { render, screen } from "@testing-library/react";
+import { composeStory } from "@storybook/react";
+import { axe } from "jest-axe";
+import Meta, {
+  ShellAuthenticated,
+  ShellAuthenticatedRedesign,
+} from "./Shell.stories";
+
+describe("ShellAuthenticated", () => {
+  it("passes the axe accessibility test suite", async () => {
+    const ShellComponent = composeStory(ShellAuthenticated, Meta);
+    const { container } = render(<ShellComponent />);
+    expect(await axe(container)).toHaveNoViolations();
+  });
+});
+
+describe("ShellAuthenticatedRedesign", () => {
+  it("passes the axe accessibility test suite", async () => {
+    const ShellComponent = composeStory(ShellAuthenticatedRedesign, Meta);
+    const { container } = render(<ShellComponent />);
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("shows the “Update scan info” navbar item", async () => {
```

Comment:
```suggestion
  it("shows the “Update scan info” navbar item when the flag is enabled", async () => {
```

Reply:
And maybe it should check that "Edit your info" is not present, and vice versa with the flag disabled?

Reply:
I’ve added a test case for testing the disabled flag [here](https://github.com/mozilla/blurts-server/pull/5671/files#diff-7a0079dacd2a1ec2e9b9deaae7d57ab8f28aecf6aea576a8575fcb7082a2fbabR29-R71).